test/models/sponsor_group_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class SponsorGroupTest < ActiveSupport::TestCase
+  test 'creating a sponsor group creates an associated application' do
+    sponsor_group = create_sponsor
+    assert_not_nil sponsor_group.application
+  end
+
+  test 'creating a sponsor group emails admin' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.